### PR TITLE
[Compute LookAhead]: Add blurb for CWP-34539

### PR DESCRIPTION
**Pull request by divyabhushan-okta**
Add blurb for CWP-34539

### CODE REVIEW

File: cspm/rn/prisma-cloud-compute-release-information/look-ahead-planned-updates-prisma-cloud-compute.adoc [hunk 1]
```diff
@@ -72,10 +72,10 @@ These IP addresses will be added to the https://docs.paloaltonetworks.com/prisma
```
Comment (divyabhushan-okta) [inline cspm/rn/prisma-cloud-compute-release-information/look-ahead-planned-updates-prisma-cloud-compute.adoc:71]: @rcaballeromx @Pubs-MV There is no blurb for this. Maybe we should comment this out?

Comment (Pubs-MV) [reply cspm/rn/prisma-cloud-compute-release-information/look-ahead-planned-updates-prisma-cloud-compute.adoc:71]: are you referring to CWP-47766, as having no blurb? @divyabhushan

**Review by Pubs-MV — COMMENTED** [review]
@rcaballeromx @divyabhushan please confirm the changes, I have to republish for the date change from June 18 to 25

File: cspm/rn/prisma-cloud-compute-release-information/look-ahead-planned-updates-prisma-cloud-compute.adoc (hunk 1 above)
Comment (divyabhushan-okta) [reply cspm/rn/prisma-cloud-compute-release-information/look-ahead-planned-updates-prisma-cloud-compute.adoc:71]: @Pubs-MV, Yes

**Review by rcaballeromx — APPROVED** [review]

**Pull request merged**